File: java/src/main/java/itp/itp1_10/ITP1_10_A.java
package itp.itp1_10;

import java.util.Scanner;

public class ITP1_10_A {

  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);

    double px = sc.nextDouble();
    double py = sc.nextDouble();
    double qx = sc.nextDouble();
    double qy = sc.nextDouble();

    System.out.println(Math.sqrt(Math.pow(px-qx, 2) + Math.pow(py-qy, 2)));
  }
}

File: kotlin/src/itp/itp1_10/ITP1_10_A.kt
package itp.itp1_10

fun dist(p: Double, q: Double): Double = Math.abs(Math.pow(p-q, 2.0))

fun main(args: Array<String>) {
    val (x1, y1, x2, y2) = readLine()!!.split(" ").map(String::toDouble)
    println(Math.sqrt(dist(x1, x2) + dist(y1, y2)))
}